.gitignore

```diff
@@ -5,3 +5,5 @@ spec/fixtures
 .vagrant
 .bundle
 vendor
+.ruby-version
+Gemfile.lock
```

Comment:
It doesn't seem to be versioned, so it wouldn't change anything. `rm Gemfile.lock` is [executed on Travis ](https://github.com/tubemogul/puppet-aerospike/blob/master/.travis.yml#L4) as well, so the test shouldn't be affected.